packages/clerk-js/src/ui/components/AccountlessPrompt/index.tsx

```diff
@@ -118,52 +124,359 @@ export const _AccountlessPrompt = (props: AccountlessPromptProps) => {
+
+            background: linear-gradient(180deg, rgba(0, 0, 0, 0) 30.5%, rgba(0, 0, 0, 0.05) 100%), #636363;
+            box-shadow:
+              0px 0px 0px 1px rgba(255, 255, 255, 0.04) inset,
+              0px 1px 0px 0px rgba(255, 255, 255, 0.04) inset,
+              0px 0px 0px 1px rgba(0, 0, 0, 0.12),
+              0px 1.5px 2px 0px rgba(0, 0, 0, 0.48),
+              0px 0px 4px 0px rgba(243, 107, 22, 0) inset;
+
+            animation: small-btn-glow 3.6s infinite;
+            animation-play-state: running !important;
+
+            ${isExpanded &&
+            ` right: 0.75rem;
+              bottom: 0.75rem;
+              width: 14.625rem;
+              color: #FDE047;
+              border-radius: 0.375rem;
+              background: linear-gradient(180deg, rgba(0, 0, 0, 0) 30.5%, rgba(0, 0, 0, 0.05) 100%), #454545;
+              animation: expanded-btn-glow 2s infinite;
+              box-shadow:
+                0px 0px 3px 0px rgba(253, 224, 71, 0) inset,
+                0px 0px 0px 1px rgba(255, 255, 255, 0.04) inset,
+                0px 1px 0px 0px rgba(255, 255, 255, 0.04) inset,
+                0px 0px 0px 1px rgba(0, 0, 0, 0.12),
+                0px 1.5px 2px 0px rgba(0, 0, 0, 0.48);
+
+            `}
+
+            @keyframes expanded-btn-glow {
+              0%,
+              100% {
+                box-shadow:
+                  0px 0px 3px 0px rgba(253, 224, 71, 0) inset,
+                  0px 0px 0px 1px rgba(255, 255, 255, 0.04) inset,
+                  0px 1px 0px 0px rgba(255, 255, 255, 0.04) inset,
+                  0px 0px 0px 1px rgba(0, 0, 0, 0.12),
+                  0px 1.5px 2px 0px rgba(0, 0, 0, 0.48);
+              }
+              50% {
+                box-shadow:
+                  0px 0px 6px 0px rgba(253, 224, 71, 0.24) inset,
+                  0px 0px 0px 1px rgba(255, 255, 255, 0.04) inset,
+                  0px 1px 0px 0px rgba(255, 255, 255, 0.04) inset,
+                  0px 0px 0px 1px rgba(0, 0, 0, 0.12),
+                  0px 1.5px 2px 0px rgba(0, 0, 0, 0.48);
+              }
+            }
+
+            @keyframes small-btn-glow {
+              0%,
+              5%,
+              70% {
+                box-shadow:
+                  0px 0px 4px 0px rgba(243, 107, 22, 0) inset,
+                  0px 0px 0px 1px rgba(255, 255, 255, 0.04) inset,
+                  0px 1px 0px 0px rgba(255, 255, 255, 0.04) inset,
+                  0px 0px 0px 1px rgba(0, 0, 0, 0.12),
+                  0px 1.5px 2px 0px rgba(0, 0, 0, 0.48);
+              }
+
+              30% {
+                box-shadow:
+                  0px 0px 6px 0px #fde047 inset,
+                  0px 0px 0px 1px rgba(255, 255, 255, 0.04) inset,
+                  0px 1px 0px 0px rgba(255, 255, 255, 0.04) inset,
+                  0px 0px 0px 1px rgba(0, 0, 0, 0.12),
+                  0px 1.5px 2px 0px rgba(0, 0, 0, 0.48);
+              }
+            }
+          `}
+        >
+          Claim keys
```

Comment:
This text probably needs to be using localization (?)

Reply:
I'm asked about this in the project channel, for now I would not localize this one.